contracts/staker/curve/ConvexTokenStaker.sol

```diff
@@ -21,6 +21,28 @@ abstract contract ConvexTokenStaker is BorrowStaker {
         _initialize(_coreBorrow, erc20Name, erc20Symbol);
     }
 
+    /// @inheritdoc ERC20Upgradeable
+    function _afterTokenTransfer(
+        address from,
+        address,
+        uint256 amount
+    ) internal override {
+        // Stake on Convex if it is a deposit
+        if (from == address(0)) {
+            // Deposit the Curve LP tokens into the convex contract and stake
+            _changeAllowance(asset(), address(_convexBooster()), amount);
+            _convexBooster().deposit(poolPid(), amount, true);
```

Comment:
this one can't be put in common as `deposit()`gets different arguments in. There is an additional `bool stake` on mainnet